src/main/kotlin/it/pagopa/wallet/domain/wallets/Wallet.kt

```diff
@@ -37,11 +36,9 @@ import java.time.Instant
  */
 @AggregateRoot
 data class Wallet(
-    @AggregateRootId val id: WalletId,
+    @AggregateRootId val walletId: WalletId,
     val userId: UserId,
     var status: WalletStatusDto,
-    val creationDate: Instant,
```

Comment:
these two fields are parts of the aggregate, they cannot be deleted, furthermore we can consider also inserting the version field in the aggregate